charts/iter8lib/templates/_metrics-common-istio.tpl

```diff
@@ -0,0 +1,116 @@
+{{- define "metrics.common.istio" }}        
```

Comment:
I am confused by the need for this. We seem to be extracting the variables from the dictionary `,Values` and added them to the dictionary `.`. We are then using them in templates directly. Why do we need to do this?

Reply:
This template is used to reduce replication in the metrics template. For example it is used 8 times in the Istio metrics template.

https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L65
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L86
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L107
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L121
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L145
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L160
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L183
https://github.com/iter8-tools/iter8/blob/963bb43f3dcf6580fd2fc9e1ea6d3066cef67fa7/charts/iter8lib/templates/_metrics-istio.tpl#L197

Let me know if I am misunderstanding your question.